buildutils/attr_parser.py

```diff
@@ -90,762 +94,191 @@ def match(self, *args):
             return False
 
 
-def fileappend(line):
+def fileappend(prop_type, line):
     """
```

Comment:
let us add a docstring describing the parameters

Reply:
The parser is hardly worth spending any time documenting. The code is small, self explanatory, and is seldom updated